apps/geo/serializers.py

```diff
@@ -50,7 +50,7 @@ class RegionSerializer(RemoveNullFieldsMixin,
 
     class Meta:
         model = Region
-        exclude = ('geo_options',)
+        exclude = ('geo_options', 'is_published')
```

Comment:
On second thought, remove is_published from here and add it as a read-only boolean field in the serializer.